docs/LOCAL_DEVELOPMENT.md

```diff
@@ -1,21 +1,26 @@
 ## Local Development
 
 ### Pre-Requisites
+The following dependencies must be installed. Please refer to your operating system, how to install them.
+
+> **Note:** For Windows 10, we recommend enabling Windows Subsystem for Linux (WSL) and installing Linux distribution of your choice,
+> for example, here are the instructions on how to install [Ubuntu](https://ubuntu.com/tutorials/ubuntu-on-windows).
 
-The following dependencies must be installed:
 - Python >=3.8 and pip
-- pre-commit
-- hugo
+- VirtualEnv
+- Hugo
 - Go
```

Comment:
Not related to the changes of the PR, but why do we need Go as a pre-req?

Reply:
The Go is required for Rain. We use rains `cfn-format` to format templates for consistency.

Reply:
I forgot about rain.
I think the rain binary should work without the go tooling, unless we are building it from source.
If we're just installing rain using `brew` then I believe can remove the requirement for Go

Reply:
We are actually not installing `Rain` globally at all. It is only part of the `pre-commit`. `Rain` will fail to run as part of `pre-commit`, if you don't have Go on your system. At least that is my experience when I was trying to use `Rain` without having `Go` installed system wide. 

Maybe instead of installing Go we can install rain globally? However if you will not use `brew`, then you need to install Go anyway. Thoughts?

Reply:
Interesting. I guess pre-commit will build it from source then.
Sorry for the distraction

Reply:
Absolutely valid point. Yes, I think `pre-commit` does build from source.